lytical/roster.json

```diff
@@ -148,7 +148,14 @@
       "materials": [
         {
             "type": "cv"
-        }
+        },
+        {
+        "type": "video", 
+        "url": "M15JTDrim5w"
+      },
+        { "type": "portfolio",
+          "url": "" 
+      },
```

Comment:
Be careful about your indentation and I cannot approve your PR if there is no portfolio link

Reply:
Hello Romeo, I have got my portfolio link included now. Enjoy :)